pycryptobot/DTO/EventInterface.py

```diff
@@ -0,0 +1,14 @@
+
+
+class EventInterface:
+    name: str
+
+    """Export to json function"""
+    def reprJSON(self):
```

Comment:
to be more in line with the code style, maybe `def repr_json`?

Reply:
no problem, I'll change

Reply:
done